sklearn/ensemble/_bagging.py

```diff
@@ -900,10 +900,10 @@ class BaggingRegressor(RegressorMixin, BaseBagging):
         processors. See :term:`Glossary <n_jobs>` for more details.
 
     random_state : int, RandomState instance, default=None
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+        Controls the randomness that generates the random subsets of the
+        original dataset (sample wise and feature wise).
+        Pass an int for reproducible output across multiple function calls.
+        See :term:`Glossary <random_state>`.
```

Comment:
I think this PR is a good opportunity to also improve the docstring of `max_features` and `max_samples` accordingly.

My understanding (though I have not checked the code) is that:

- `max_samples` controls "sample-wise" resampling **with replacement**, which means that is a always random.
- `max_features` controls "feature-wise" sampling **without replacement**, which means that the default `max_features=1.` is deterministic (no feature-wise randomness).

Maybe we could also mention that the `random_state` attribute (if any) of the `estimators_` generated from `base_estimator` are independently seeded but are all dependent on the main `random_state` value.